plugins/active_directory_ldap/komand_active_directory_ldap/util/utils.py

```diff
@@ -133,30 +136,48 @@ def find_parentheses_pairs(query_string: str) -> dict:
         return pairs
 
     @staticmethod
-    def escape_brackets_for_query(query: str, pairs: dict) -> str:
+    def find_characters_in_string(
+        input_list_of_characters: List[str],
+        input_string: str,
+        start_index: Optional[int] = None,
+        end_index: Optional[int] = None,
+    ) -> bool:
+        """
+        This method will check if input_string contains any characters from a input_list_of_characters
+        :param input_list_of_characters: List of characters to search for
+        :param input_string: The string to evaluate
+        :param start_index: Start index of input_string to begin searching characters for
+        :param end_index: End index of input_string to begin searching characters for
+        :return: Bool value indicates whether input_string contains any of characters (True) or not (False)
+        """
+        for character in input_list_of_characters:
+            if input_string.find(character, start_index, end_index) >= 0:
+                return True
+        return False
+
+    @staticmethod
+    def escape_brackets_for_query(query: str) -> str:
         """
         This method will properly escape a query
         :param query: The string to evaluate
         :param pairs: indexes of the start and end of brackets
```

Comment:
Is pairs still needed in the description if it is not included anymore?
Possible change: Update docstring to reflect new changes